### ISSUE TITLE
--outcmd equivalent for all verbs

### ISSUE THREAD

**Issue by 3tilley** [Canop/broot#773]
Hello,

I'm loving broot.

I'm hacking around on it, both my config and the code inside. `--outcmd` is great for debugging, as well as the obvious usage for br.

I think there would be some use for an equivalent for verbs executed inside broot. A file that gathers everything that was executed inside, so someone can see the effects of their verbs.

I think this could be especially useful for anyone writing their own verbs, and for that reason I would propose it as a CLI command - rather than just an env flag targetted at broot devs. What about --outverb? `--verb-out(put)` would probably be my preference, but consistency probably trumps.

If you agree, I'll start working on it.

**Comment by Canop** [comment]
I'm not sure I understand your use case. Is that some kind of log ?

**Comment by 3tilley** [comment]
Yes basically, I guess it could also be used to mark a load of files in the editor and come back to them later, but it's mainly for debugging verbs.

The reason I would say it's distinct from just using the log is that i get the impression that users are encouraged to hack around and create their own verbs, and this would make it a lot easier. If it was only for people hacking on broot itself I'd say they could use the log

**Comment by Canop** [comment]
Kind of related: https://github.com/Canop/broot/issues/825

**Comment by Canop** [comment]
Two things cover this now. To see what your verbs execute, run broot with `BROOT_LOG=info` (or `debug`): every applied command and every launched program is written to broot.log. To collect paths or anything else from inside broot into a file, use `--verb-output` with the `:write_output` internal (since 1.34, #825). Closing.
